openmc/deplete/__init__.py

```diff
@@ -6,10 +6,24 @@
 """
 
 from .dummy_comm import DummyCommunicator
+
 try:
     from mpi4py import MPI
+
     comm = MPI.COMM_WORLD
     have_mpi = True
+    # check if running with MPI and if hdf5 is MPI-enabled
+    from h5py import get_config
```

Comment:
This import can/should be up top before any code logic. The only import we want to protect in the try/except block is for mpi4py. When you move it, also note that PEP8 specifies an [ordering for imports](https://www.python.org/dev/peps/pep-0008/#imports).